share/pot/create.sh

```diff
@@ -557,7 +557,7 @@ pot-create()
 					fi
```

Comment:
The same comments apply as above (condensing the logic, not showing help if the file can't be opened etc). Shouldn't "_dns" also be set to "inherit" explicitly in  line 543?

```
				case $OPTARG in
					inherit|pot|off)
						_dns="$OPTARG"
						;;

```

Reply:
In create, _dns has "inherit" as default value, but it doesn't hurt to reassign it :)
In clone, the default value is the one in the cloned pot
Reworking it